NOnion/Cells/CellVersions.fs

```diff
@@ -15,18 +15,12 @@ type CellVersions =
             if (reader.BaseStream.Length - reader.BaseStream.Position) % 2L
                <> 0L then
                 failwith
-                    "Version packet payload is invalid, payload length should be divisible by 2"
+                    "Version packet payload is invalid, payload length should be divisable by 2"
```

Comment:
@symboliq why are you re-introducing a typo that I fixed?

Reply:
Another accidental merge issue.